src/network.ts

```diff
@@ -8,9 +8,12 @@ const AFTER = 1;
 
 export class Network {
   public readonly addr = new Address();
+  public readonly network?: string;
   private netbits = -1;
 
   public constructor(network?: string, throwErrors?: boolean) {
+    this.network = network;
```

Comment:
I guess they don't do this to avoid memory bloat. What does the memory look like if we load all ranges into memory? I guess it isn't too bad?

Reply:
I haven't checked but in our case the ranges are ~3000 strings, all of which will already be in memory because we load in the whole lot and parse it as JSON. I don't expect this to cause any problems.